tests/test_pbh.py

```diff
@@ -0,0 +1,795 @@
+
+        def getPbhRuleCount(self):
+            return self.ruleCount
+
+        def incPbhTableCount(self):
+            self.tableCount += 1
+
+        def decPbhTableCount(self):
+            self.tableCount -= 1
+
+        def getPbhTableCount(self):
+            return self.tableCount
+
+    class LagRefCountHelper(object):
+        def __init__(self):
+            self.lagCount = 0
+            self.lagMemberCount = 0
+
+        def incLagCount(self):
+            self.lagCount += 1
+
+        def decLagCount(self):
+            self.lagCount -= 1
+
+        def getLagCount(self):
+            return self.lagCount
+
+        def incLagMemberCount(self):
+            self.lagMemberCount += 1
+
+        def decLagMemberCount(self):
+            self.lagMemberCount -= 1
+
+        def getLagMemberCount(self):
+            return self.lagMemberCount
+
+    def strip_prefix(self, s, p):
+        return s[len(p):] if s.startswith(p) else s
+
+    @pytest.fixture(autouse=True)
+    def pbh_ref_count(self):
+        pbhlogger.info("Create PBH reference count helper")
+        yield self.PbhRefCountHelper()
+        pbhlogger.info("Remove PBH reference count helper")
+
+    @pytest.fixture(autouse=True)
+    def lag_ref_count(self):
+        pbhlogger.info("Create LAG reference count helper")
+        yield self.LagRefCountHelper()
+        pbhlogger.info("Remove LAG reference count helper")
+
+    @pytest.fixture(autouse=True)
+    def pbh_port_channel_0001(self, lag_ref_count):
+        try:
+            meta_dict = {
+                "name": "PortChannel0001",
+                "member": "Ethernet120"
+            }
+
+            lag_id = self.strip_prefix(meta_dict["name"], "PortChannel")
+
+            pbhlogger.info("Create LAG: {}".format(meta_dict["name"]))
+            self.dvs_lag.create_port_channel(lag_id)
+            lag_ref_count.incLagCount()
+            self.dvs_lag.get_and_verify_port_channel(lag_ref_count.getLagCount())
+
+            pbhlogger.info("Create LAG member: {}".format(meta_dict["member"]))
+            self.dvs_lag.create_port_channel_member(lag_id, meta_dict["member"])
+            lag_ref_count.incLagMemberCount()
+            self.dvs_lag.get_and_verify_port_channel_members(lag_ref_count.getLagMemberCount())
+
+            yield meta_dict
+
+        finally:
+            pbhlogger.info("Remove LAG member: {}".format(meta_dict["member"]))
+            self.dvs_lag.remove_port_channel_member(lag_id, meta_dict["member"])
+            lag_ref_count.decLagMemberCount()
+            self.dvs_lag.get_and_verify_port_channel_members(lag_ref_count.getLagMemberCount())
+
+            pbhlogger.info("Remove LAG: {}".format(meta_dict["name"]))
+            self.dvs_lag.remove_port_channel(lag_id)
+            lag_ref_count.decLagCount()
+            self.dvs_lag.get_and_verify_port_channel(lag_ref_count.getLagCount())
+
+    @pytest.fixture(autouse=True)
+    def pbh_port_channel_0002(self, lag_ref_count):
```

Comment:
Suggestion: can we create meta helpers + parametrization to avoid some code duplication acros fixutres?